src/system/Wizard/Wizard.tsx

```diff
@@ -0,0 +1,62 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * External dependencies
+ */
+import React, { useLayoutEffect, useState } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { Box, WizardStep } from '..';
+import { WizardStepProps } from './WizardStep';
+
+export interface WizardProps {
+	steps: WizardStepProps[];
+	activeStep?: number;
+	completed?: number[];
+	skipped?: number[];
+	className?: string;
+	titleAutofocus?: boolean;
+}
+export const Wizard = React.forwardRef< HTMLDivElement, WizardProps >(
+	(
+		{ steps, activeStep, completed = [], skipped = [], className = null, titleAutofocus = false },
+		forwardRef
+	) => {
+		const [ didMount, setDidMount ] = useState( false );
+		const [ initialStep ] = useState( activeStep );
+		// didMount helps us to track the initial render, so we can focus the title only subsequent renders
+		// to avoid stealing the focus from the page we're in.
+		useLayoutEffect( () => {
```

Comment:
i know this is not new code, but this is confusing and duplicative. seems like a single layouteffect in WizardStep should be enough

Reply:
@chriszarate I agree on this, but I need some more time to test this out properly (I remember it was an edge case so I want to be sure it's working as expected).

If you're ok with this I'll proceed with merging this PR so we can have all the extra components converted and get back at it in a separate one.

Reply:
Fine with me to merge